Bottom: 7e9a8aa7fe56b9d12a711d675706fe3506184eee
Top:    f92d03eb86bdc21c17723f036529ab112af02b04
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-07-31 14:15:51 +0100

Refresh of extend-board-export

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index c4de6b7..2efc341 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -60,7 +60,7 @@
 
 
 #define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
-#define HACK_MASK_THICKNESS MM_TO_COORD(0.05)
+#define HACK_MASK_THICKNESS MM_TO_COORD(0.01)
 
 static GList *object3d_test_objects = NULL;
